### PR TITLE
Apply to expiration time item information popup

### PR DESCRIPTION
**Pull request by hskim881028**
### Description

SSIA

### Required Reviewers

@planetarium/9c-dev , @Namyujeong 

### Screenshot

ezgif com-gif-maker (1)

### CODE REVIEW

File: nekoyume/Assets/_Scripts/UI/Model/ItemInformationTooltip.cs [hunk 1]
```diff
@@ -75,14 +65,9 @@ public override void Dispose()
             SubmitButtonEnabledFunc.Dispose();
             SubmitButtonEnabled.Dispose();
             SubmitButtonText.Dispose();
-            PriceEnabled.Dispose();
             Price.Dispose();
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/UI/Model/ItemInformationTooltip.cs:68]:
```suggestion
            Price.Dispose();
            ExpiredBlockIndex.Dispose();
```

File: nekoyume/Assets/_Scripts/UI/Module/Button/SubmitButton.cs [hunk 2]
```diff
@@ -1,3 +1,4 @@
+using System;
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/UI/Module/Button/SubmitButton.cs:1]:
```suggestion
```

**Review by boscohyun — CHANGES_REQUESTED** [review]

**Review by boscohyun — APPROVED** [review]
There is conflict.😂

**Review by Namyujeong — CHANGES_REQUESTED** [review]
The remaining time is strange. If it's 16000 blocks, it should be about 67 hours when calculated as 15 seconds per block.

image

**Comment by hskim881028** [comment]
> The remaining time is strange. If it's 16000 blocks, it should be about 67 hours when calculated as 15 seconds per block.
> 
> image

Modifications completed!

**Review by Namyujeong — APPROVED** [review]

**Pull request merged**